# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/nikolaus/dnrd/instopt.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/Makefile.org	1999-11-03 01:47:10.000000000 +0100
+++ ./src/Makefile	2003-04-06 21:52:34.000000000 +0200
@@ -6,8 +6,8 @@
 CFLAGS		= ${CWARN} ${COPT} ${CDEF} ${CDBG}
 LDFLAGS		= 
 LIBS            = -lpthread
-INSTDIR		= /usr/local/sbin
-MANDIR		= /usr/local/man/man8
+INSTDIR		= /usr/sbin
+MANDIR		= /usr/man/man8
 SHELL		= /bin/sh
 
 all:	dnrd
@@ -18,8 +18,8 @@
 
 install: dnrd
 	strip dnrd
-	install -m 0751 -o root -g root -s dnrd ${INSTDIR}
-	install -m 0644 -o root -g root -s ../doc/dnrd.8 ${MANDIR}
+	install -m 0751 -o root -g root dnrd ${INSTDIR}
+	install -m 0644 -o root -g root ../doc/dnrd.8 ${MANDIR}
 
 sources = args.c cache.c common.c dns.c lib.c main.c master.c \
 	  query.c relay.c sig.c tcp.c udp.c
